sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubProducer.cs

```diff
@@ -344,6 +373,26 @@ protected EventHubProducer()
         [EditorBrowsable(EditorBrowsableState.Never)]
         public override string ToString() => base.ToString();
 
+        private DiagnosticScope CreateDiagnosticScope()
+        {
+            DiagnosticScope scope = _clientDiagnostics.CreateScope("Azure.Messaging.EventHubs.EventHubProducer.Send");
+            scope.AddAttribute("kind", "producer");
```

Comment:
I get the feeling that we'll see these keys elsewhere; may be worth considering hoisting them into the diagnostics class as `const` members.